docs/scripting/trust scripts/sys.status.mdx

```diff
@@ -6,18 +6,52 @@ title: sys.status
 
 sys.status returns the status of the user that runs it.
 
+### Security Level
+
+HIGHSEC
+
 ## Syntax
 
+### CLI
+
+```
 sys.status
+```
+
+### Script
+
+```
+#fs.sys.status()
+```
 
 ### Parameters
 
-There are no known parameters
+There are no known parameters.
 
 ### Return
 
-It returns status, i need to get an actual output
+sys.specs returns a `String`
```

Comment:
I'd write the type as follows:
```suggestion
sys.specs returns a string
```

Otherwise there might be some confusion about `String` vs. the primitive string type